score/score.h

```diff
@@ -21,15 +30,37 @@ int watermelon;
 */
 void score_init(struct Score *scoreStart);
 
+/**
+ * Difficulty fucntions that assigned associated difficulty multiplier value
+ * to multiplier (default value is 1): easy = 1, medium = 2, hard = 3
+ * @param mode make call to struct Score multiplier to assign the diffiulty
+ * multiplier value
+ */
 void easy(struct Score *mode);
 void medium(struct Score *mode);
```

Comment:
Even though all these methods do the same thing, doxygen isn't intelligent enough to associate this comment with all of them. You'll have to copy / paste the description for each of these.